dimagi/utils/couch/undo.py

```diff
@@ -32,3 +35,13 @@ def soft_delete(self, domain_included=True):
             record.save()
             self.save()
             return record
+
+
+def is_deleted(doc):
+    """
+    Guess if a document is deleted. Returns False if we're not sure.
```

Comment:
that makes sense. how about just tweaking the comment? i don't feel that strongly about changing the name, more thinking that if i were a brand-new dev and read that comment, I'd think, "Wait, is there some other indication that a document could b deleted, do they also use is_deleted flags or something?"